modules/plottinggl/include/modules/plottinggl/processors/parallelcoordinates/parallelcoordinates.h

```diff
@@ -109,11 +109,15 @@ class IVW_MODULE_PLOTTINGGL_API ParallelCoordinates : public Processor {
                     const std::vector<ParallelCoordinatesAxisSettingsProperty *> &enabledAxis);
 
     void updateBrushing();
+    std::unordered_set<size_t> brushedID;
 
     DataInport<DataFrame> dataFrame_;
     BrushingAndLinkingInport brushingAndLinking_;
     ImageOutport outport_;
 
+    BoolProperty brushOnlyWhenMouseRelease_;
+    bool mouseReleased = true;
```

Comment:
mouseReleased_
The implementation also support touch :)
I think that this variable could be removed with some refactoring. See later comment